OpenRA.Mods.Common/Traits/BotModules/BotModuleLogic/BaseBuilderQueueManager.cs

```diff
@@ -151,6 +152,16 @@ bool TickQueue(IBot bot, ProductionQueue queue)
 						location = possibleBuilding.Actor.Location + possibleBuilding.Trait.Info.Offset;
 					}
 				}
+				else if (lineBuildInfo != null)
```

Comment:
This is only used once, so it would make sense to change this to `else if (actorInfo.HasTraitInfo<LineBuildInfo>())`.